### PR TITLE
Add logic to Calculator

### PR DESCRIPTION
**Pull request by nachosala89**
- Added files calculate.js and operate.js to implement the math logic.
- Added state to Calculator.js.
- Added event handlers to buttons.
- Implemented a feature to display the results.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,42 +1,63 @@
 import React from 'react';
 import Button from './Button';
```
Comment (vichuge) [inline src/components/Calculator.js:2]: - [x] Here, inside Calculator.js is where you could import the calculte.js file, please do this. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (nachosala89) [reply src/components/Calculator.js:2]: Sorry, but why would I have to import the calculate.js file there if I don't use it in Calculator.js? I use the calculate function in Button.js

**Comment by Davidosky007** [comment]
## Hi @nachosala89 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
nice-job



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**